### DISCUSSION TITLE
Welcome to OpenMetadata Discussions!

### DISCUSSION THREAD

**Discussion by harshach** [open-metadata/OpenMetadata#541] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robles-dec** [comment]
Hello,
I am completely new to openmetadata.  
So basically, I do not quite get the concept of openmetadata, what are its primary use, and use-cases.  

I like to be able to use metadata that could be extracted from file / objects to create some form of database that could provide a catalog of all of the metadata found in a big collection of files.  Additionaly, maintaining lineage and provenance on the data.  I know this is pretty generic, but I like to find out if this tool could provide this basic functionality.


Thanks in advance.

**Comment by jgnf01** [comment]
I work with infrastructure and I'm in this project to update openmetada


I have version 1.0.1 and I need to go to version 1.6.0, I would like to know if I need to go from 01 to 01 or can I update directly?

**Comment by jgnf01** [comment]
I have version 1.0.1 and I need to go to version 1.6.0, I would like to know if I need to go from 01 to 01 or can I update directly?
Can you give me a script to do this update?
